Dockerfile

```diff
@@ -9,16 +9,20 @@ RUN mkdir /var/run/sshd
 RUN echo "REPREPRO_BASE_DIR=/data/debian" > /etc/environment
 
 # Configure an reprepro user (admin)
-RUN adduser --system --group --shell /bin/bash --uid 600 --disabled-password --no-create-home reprepro
+RUN adduser --system --group --shell /bin/bash --uid 600 --disabled-password --home=/data reprepro
 
 # Configure an apt user (read only)
-RUN adduser --system --group --shell /bin/bash --uid 601 --disabled-password --no-create-home apt
+RUN adduser --system --group --shell /bin/bash --uid 601 --disabled-password --home=/data apt
```

Comment:
that helped me because i was able to log in by ssh to check what's accessible, i agree it's strange to share home folder anyway